### PR TITLE
Updated usage of ApplicationStartInfo to BinaryInfo

### PR DESCRIPTION
**Pull request by dhruv-vora**
**Description:** 
- Changed usage of ApplicationStartInfo to BinaryInfo
- Changed ExeName to Command

### CODE REVIEW

File: exporter/awsemfexporter/cwlog_client.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func newCloudWatchLogClient(svc cloudwatchlogsiface.CloudWatchLogsAPI, logger *z
 }
 
 // NewCloudWatchLogsClient create cloudWatchLogClient
-func NewCloudWatchLogsClient(logger *zap.Logger, awsConfig *aws.Config, startInfo component.ApplicationStartInfo, sess *session.Session) LogClient {
+func NewCloudWatchLogsClient(logger *zap.Logger, awsConfig *aws.Config, startInfo component.BinaryInfo, sess *session.Session) LogClient {
```
Comment (Aneurysm9) [inline exporter/awsemfexporter/cwlog_client.go:61]:
```suggestion
func NewCloudWatchLogsClient(logger *zap.Logger, awsConfig *aws.Config, binaryInfo component.BinaryInfo, sess *session.Session) LogClient {
```

File: internal/version/version.go [hunk 2]
```diff
@@ -16,6 +16,3 @@ package version
 
 // Version variable will be replaced at link time after `make` has been run.
 var Version = "latest"
-
-// GitHash variable will be replaced at link time after `make` has been run.
-var GitHash = "<NOT PROPERLY GENERATED>"
```
Comment (Aneurysm9) [inline internal/version/version.go:21]: Should this be removed here?  Even if it isn't used I'd probably prefer to remove it in a separate PR to maintain the focus of this PR on `ApplicationStartInfo` -> `BinaryInfo`.

File: exporter/awsemfexporter/cwlog_client.go (hunk 1 above)
Comment (dhruv-vora) [reply exporter/awsemfexporter/cwlog_client.go:61]: Sure, I will rename all the variables from `startInfo` to `binaryInfo`

File: internal/version/version.go (hunk 2 above)
Comment (dhruv-vora) [reply internal/version/version.go:21]: I will add this back

**Review by Aneurysm9 — APPROVED** [review]

**Review by rakyll — APPROVED** [review]

**Pull request closed without merging**